src/components/publiq-ui/Button.js

```diff
@@ -8,12 +10,32 @@ const ButtonVariants = {
   DANGER: 'danger',
 };
 
+const getValue = (path) => (props) =>
+  getValueFromTheme(props, `components.button.${path}`);
```

Comment:
I think we can change the function so it becomes

```js
const getValue =
  getValueFromTheme(`components.button`);
```

here

Reply:
I would have to change `getValueFromTheme` first tho, or not?

Reply:
with 'change the function' I meant that, yes